src/stream/src/from_proto/mview.rs

```diff
@@ -59,6 +59,7 @@ impl ExecutorBuilder for MaterializeExecutorBuilder {
     }
 }
 
+/// TODO: remove it
```

Comment:
Is this supposed to be done in this or future PR?

```suggestion
// TODO: remove it
```